pkg/spec/cluster.go

```diff
@@ -193,6 +193,8 @@ type ClusterStatus struct {
 
 	// Size is the current size of the cluster
 	Size int `json:"size"`
+	// HealthySize is the number of pods passing the liveliness probe, HTTP GET on port 2379
```

Comment:
I like names better. users can inspect the pod state with k8s api as long as they have the pod name.